test/integration/scenarios/stress/4.register_multisignature.js

```diff
@@ -74,19 +71,19 @@ module.exports = function(params) {
 		});
 
 		describe('sending multisignature registrations', () => {
-			var signatures = [];
-			var agreements = [];
-			var numbers = _.range(maximum);
-			var i = 0;
-			var j = 0;
+			const signatures = [];
+			let agreements = [];
+			const numbers = _.range(maximum);
+			let i = 0;
+			let j = 0;
 
 			before(() => {
 				transactions = [];
 				return Promise.all(
```

Comment:
Would be nice to have a comment what is happening here.

Reply:
This one should be easy. As in every other test, we prepare the accounts transferring some Lisk to make them able to create more transactions. @4miners Is it not straightforward? If I add the comment here, we should add it to every single file of every test suite.